Drop-In/build.gradle

```diff
@@ -60,7 +60,7 @@ android {
 dependencies {
     api 'com.braintreepayments.api:braintree:3.15.0'
     api 'com.braintreepayments:card-form:5.0.0'
-    api 'com.braintreepayments.api:three-d-secure:3.14.2'
+    api 'com.braintreepayments.api:three-d-secure:3.15.0'
```

Comment:
Should we call this out in the CHANGELOG?

Reply:
Typically we have only been calling out bumping `braintree_android` version in the CHANGELOG. This update is just to align the 3DS dependency with the core dependency that was bumped in the last release. We could add it as a separate CHANGELOG item though?

Reply:
Hmm... Is there a reason why we didn't bump the `three-d-secure` version when we bumped the `braintree` version? I don't have strong feelings about adding a CHANGELOG entry, but it seems like it couldn't hurt? 🤷‍♀️

Reply:
Yeah tbh rushing to get the Hostname Verifier fix out was the reason. The two dependencies are fundamentally the same since the `three-d-secure` went virtually untouched in the update.

Reply:
Gotcha, that totally makes sense. Yeah, I'd say it's up to you guys if you think a CHANGELOG entry would be helpful.